src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/HologramProjector.java

```diff
@@ -27,7 +28,10 @@
 
 public class HologramProjector extends SlimefunItem {
 
-    private static final String OFFSET_PARAMETER = "offset";
+    private static final String Y_OFFSET_PARAMETER = "y_offset";
+    private static final String X_OFFSET_PARAMETER = "x_offset";
+    private static final String Z_OFFSET_PARAMETER = "z_offset";
```

Comment:
Also... why is this not in order...

Reply:
Y is likely the most often used offset, but I can change it for uniformity